CDK/Serverless image

Mallear opened the pull request:
Create an image for CDK and serverless use cases.

Mallear commented:
I'm currently having some trouble with this new image, still testing.

Mallear commented:
OK for merge ! 
Need to maintains 3.7 and 3.8 versions as AWS Lambda doesn't support 3.8 yet and serverless need to be run in the same version as the lambda runtime.

May delete the 3.7 after the 1rst release.

clemstoquart commented:
@Mallear Do we really need another image ?

Do you think it would be possible to use the existing serverless image ?

Mallear commented:
> @Mallear Do we really need another image ?
> 
> Do you think it would be possible to use the existing serverless image ?

Yep, that's why I renamed serverless image and just deleted serverless as it is managed as a project dependency to have the right version for each project.

clemstoquart commented:
Ok :)

BTW you should probably use Node 12 instead of 13.

Mallear commented:
> Ok :)
> 
> BTW you should probably use Node 12 instead of 13.

More reliable ? More secure ? I will update this and rebase 👍

clemstoquart commented:
> > Ok :)
> > BTW you should probably use Node 12 instead of 13.
> 
> More reliable ? More secure ? I will update this and rebase 👍

Odd Node release are considered as cutting edge version that should only be used if you need the latest features. Staying on the last LTS release is safer.

Mallear commented:
> > > Ok :)
> > > BTW you should probably use Node 12 instead of 13.
> > 
> > 
> > More reliable ? More secure ? I will update this and rebase 👍
> 
> Odd Node release are considered as cutting edge version that should only be used if you need the latest features. Staying on the last LTS release is safer.

Good to know, thanks :)

clemstoquart commented:
I don't think the build error is related.

Mallear commented:
Fixed :D

clemstoquart commented on README.md:
```diff
@@ -183,6 +183,12 @@ https://hub.docker.com/r/ekino/ci-python/tags
+### Pynode
+https://hub.docker.com/r/ekino/ci-pynode/tags
+
+Based on Python image, contains Nodejs 13. Used for serverless and CDK use cases, with tools installed in a
```
```suggestion
Based on Python image, contains Nodejs 12. Used for serverless and CDK use cases, with tools installed in a
```

Mallear replied:
Fixed 👍
